recipe-server/client/control_new/state/action-types.js

```diff
@@ -4,6 +4,8 @@ export const ACTION_RECEIVE = 'ACTION_RECEIVE';
 export const APPROVAL_REQUEST_DELETE = 'APPROVAL_REQUEST_DELETE';
 export const APPROVAL_REQUEST_RECEIVE = 'APPROVAL_REQUEST_RECEIVE';
 export const EXTENSION_RECEIVE = 'EXTENSION_RECEIVE';
+export const EXTENSIONS_LISTING_COLUMNS_CHANGE = 'EXTENSIONS_LISTING_COLUMNS_CHANGE';
+export const EXTENSIONS_PAGE_RECEIVE = 'EXTENSIONS_PAGE_RECEIVE';
```

Comment:
Actually i think maybe I should do that in another PR that does the same for recipes once this lands.